script.js

```diff
@@ -189,33 +189,42 @@ var conveyorBelt = document.createElement('div');
 
 
 var containers = {
-    bin1: { element: document.createElement('img'), x: 100, y: 600 },
-    bin2: { element: document.createElement('img'), x: 300, y: 600 },
-    bin3: { element: document.createElement('img'), x: 600, y: 600 },
-    bin4: { element: document.createElement('img'), x: 1000, y:600 }
-}
+    bin1: { element: document.createElement('img'), x: 100, y: 600, itemIndexInBin: [] },
```

Comment:
This variable name is a tad confusing to me - could it just be something like `itemIds` or `itemIdsInBin`? Related to my previous comment, what you're storing here is the ID of the item, it just happens to be the same as the index right now. If you end up changing that later, this variable name becomes more confusing, I think.